internal/local/command_kafka_start.go

```diff
@@ -78,6 +78,7 @@ func (c *command) kafkaStart(cmd *cobra.Command, _ []string) error {
 
 	containers, err := dockerClient.ContainerList(context.Background(), types.ContainerListOptions{All: true})
 	if err != nil {
+		output.ErrPrintln(false, "The local commands have been verified to work with Docker Desktop version as late as 4.25.0.")
```

Comment:
How about including this as a suggestion? "Verify that you are not using an unverified version of Docker (versions greater than v4.25.0)."

Reply:
Also, please make a constant instead of copying and pasting code!

Reply:
yes i added a commit to constant it!
will update the string now